### ISSUE TITLE
Digital Ocean install

### ISSUE THREAD

**Issue by psychemedia** [jupyterhub/the-littlest-jupyterhub#252]
Following the docs demo for installing on Digital Ocean, I'm finding there's a time out the first time I try with a new droplet (identified by trying the install script from an ssh terminal rather than user data route).

The server doesn't start within the short time allocated to the four start attempts and crashes out?

**Comment by yuvipanda** [comment]
I always try on the smallest instance and it works. Let me try again.

**Comment by choldgraf** [comment]
@psychemedia is this one resolved for you? Maybe it was a short-term issue?

**Comment by psychemedia** [comment]
It was intermittent for me... Not tried it recently... Maybe just DO having a moment...
